src/utils/setupI13n.js

```diff
@@ -25,16 +26,11 @@ module.exports = function setupI13n (Component, options, plugins) {
     var RootI13nComponent;
     var componentName = Component.displayName || Component.name;
 
-    var reactI13n = new ReactI13n(options);
-    plugins.forEach(function setPlugin(plugin) {
-        reactI13n.plug(plugin);
-    });
-
     RootI13nComponent = React.createClass({
 
         mixins: [I13nUtils],
 
-        displayName: options.displayName || ('RootI13n' + componentName),
+        displayName: options.displayName || ('setupI13n(' + componentName + ')'),
```

Comment:
Minor:  I'd remove parenthesis from the displayName.   I like the older name better personally :)